tests/integration/cli/test_test.py

```diff
@@ -140,6 +140,8 @@ def run_gas_test(
         remainder = "\n".join(expected[actual_len:])
         pytest.fail(f"Expected contains more than actual:\n{remainder}")
 
+    print("actual:", "\n".join(actual))
```

Comment:
- [ ] delete print statements

Reply:
This was intentional instrumentation because I wasn't immediately able to repro a CI test issue.